ext/scripts/install_scripts/install_micromamba.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+# Install micromamba
+curl -L "https://micromamba.snakepit.net/api/micromamba/linux-64/$1" | tar -xvj "bin/micromamba"
+
+# Setup MAMBA_ROOT_PREFIX
+mkdir -p "$MAMBA_ROOT_PREFIX/conda-meta" && \
+chmod -R a+rwx "$MAMBA_ROOT_PREFIX" && \
+
+# Activate micromamba for the bash
+echo "source /usr/local/bin/_activate_current_env.sh" >> ~/.bashrc && \
+echo "source /usr/local/bin/_activate_current_env.sh" >> /etc/skel/.bashrc && \
+ln -s /usr/local/bin/_activate_current_env.sh /etc/profile.d/_activate_current_env.sh && \
+chmod -R a+rx /usr/local/bin/_activate_current_env.sh
```

Comment:
Do we need to add the script to profile.d?

Reply:
I more or less copied this from the mamba docker containers, but still think it is necessary, because the .bashrc is only for root and the UDFs don't run with root.